admin/tests/unit/controllers/authenticated/target-profiles/target-profile/organizations_test.js

```diff
@@ -209,16 +206,84 @@ module('Unit | Controller | authenticated/target-profiles/target-profile/organiz
     });
 
     module('when there is an error', () => {
-      test('it shows a notification', async function(assert) {
-        const errors = {};
-        controller.notifications = Service.create({ error: sinon.stub() });
-        controller.model = { targetProfile: { attachOrganizationsFromExistingTargetProfile: sinon.stub().rejects(errors) } };
-        controller.existingTargetProfile = 1;
+      module('when the error is correctly formed', () => {
+        test('it shows notification for each 404 error found', async function(assert) {
+          const errors = {
+            errors: [
+              { status: '404', detail: 'I am displayed 1' },
+              { status: '404', detail: 'I am displayed 2' },
+            ],
+          };
+          controller.notifications = Service.create({ error: sinon.stub() });
+          controller.model = { targetProfile: { attachOrganizationsFromExistingTargetProfile: sinon.stub().rejects(errors) } };
+          controller.existingTargetProfile = 1;
 
-        await controller.attachOrganizationsFromExistingTargetProfile(event);
+          await controller.attachOrganizationsFromExistingTargetProfile(event);
+
+          assert.ok(controller.notifications.error.calledWith('I am displayed 1'));
+          assert.ok(controller.notifications.error.calledWith('I am displayed 2'));
+        });
+
+        test('it shows notification for each 412 error found', async function(assert) {
+          const errors = {
+            errors: [
+              { status: '412', detail: 'I am displayed too 1' },
+              { status: '412', detail: 'I am displayed too 2' },
+            ],
+          };
+          controller.notifications = Service.create({ error: sinon.stub() });
+          controller.model = { targetProfile: { attachOrganizationsFromExistingTargetProfile: sinon.stub().rejects(errors) } };
+          controller.existingTargetProfile = 1;
+
+          await controller.attachOrganizationsFromExistingTargetProfile(event);
+
+          assert.ok(controller.notifications.error.calledWith('I am displayed too 1'));
+          assert.ok(controller.notifications.error.calledWith('I am displayed too 2'));
+        });
+
+        test('it shows notification for each 400 error found', async function(assert) {
```

Comment:
**suggestion (non-blocking) :** Ce test montre un comportement particulier pour les erreurs 400 qui n'existe pas les erreurs 400 sont traitées comme dans le test `it shows default notification if there is another error found'`